lib/api/bucketDeleteTagging.js

```diff
@@ -0,0 +1,50 @@
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const metadata = require('../metadata/wrapper');
+const util = require('node:util');
+
+/**
+ * Bucket Delete Tagging - Delete a bucket's Tagging
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+async function bucketDeleteTagging(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'bucketDeleteTagging' });
+
+    let bucket;
+    const metadataValidateBucketPromise = util.promisify(metadataValidateBucket);
+    let updateBucketPromise = util.promisify(metadata.updateBucket);
+    // necessary to bind metadata as updateBucket calls 'this', causing undefined otherwise
+    updateBucketPromise = updateBucketPromise.bind(metadata);
+    const bucketName = request.bucketName;
+    const metadataValParams = {
+        authInfo,
+        bucketName,
+        requestType: 'bucketDeleteTagging',
+    };
+
+    try {
+        bucket = await metadataValidateBucketPromise(metadataValParams, log);
+        bucket.setTags([]);
+        await updateBucketPromise(bucket.getName(), bucket, log);
+        const corsHeaders = collectCorsHeaders(request.headers.origin,
+            request.method, bucket);
+        pushMetric('deleteBucketTagging', log, {
+            authInfo,
+            bucket: bucketName,
+        });
+        return callback(null, corsHeaders);
+    } catch (err) {
+        const corsHeaders = collectCorsHeaders(request.headers.origin,
+            request.method, bucket);
+        log.error('error processing request', { error: err,
```

Comment:
same here with logging the bucket name

Reply:
done 684e92cd5c1651787050c6a7bb8e8f440cb8ce2f